src/viper/trans.ml

```diff
@@ -115,6 +115,8 @@ type ctxt =
     ghost_conc : (ctxt -> exp -> exp) list ref;
   }
 
+let str_literal_to_fld : (string, int) Hashtbl.t = Hashtbl.create 32
```

Comment:
I guess now it's not `..to_fld`. Oh, and bwt it's text literals
Maybe:
```suggestion
(* All text literals injectively mapped into int values *)
let text_literals : (string, int) Hashtbl.t = Hashtbl.create 32
```

Reply:
Done